core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BillingProfileManagerDAO.scala

```diff
@@ -124,25 +129,57 @@ class BillingProfileManagerDAOImpl(samDAO: SamDAO,
         ctx.userInfo
       )
       .flatMap {
-        case true =>
-          Future.successful(
-            Seq(
-              RawlsBillingProject(
-                RawlsBillingProjectName(azureConfig.billingProjectName),
-                CreationStatuses.Ready,
-                None,
-                None,
-                azureManagedAppCoordinates = Some(
-                  AzureManagedAppCoordinates(
-                    UUID.fromString(azureConfig.azureTenantId),
-                    UUID.fromString(azureConfig.azureSubscriptionId),
-                    azureConfig.azureResourceGroupId
+        case true => Future.successful(callListProfiles())
+        case _    => Future.successful(Seq())
+      }
+  }
+
+  def getHardcodedAzureBillingProject(samUserResourceIds: Set[String], userInfo: UserInfo)(implicit
```

Comment:
This was only using the resourceIds from Sam, so I thought it might be more clear, or make mocking a bit easier to consume just the ids. 

It turned out not to matter much.

Reply:
This method should be going away before long, so I don't think it matters.